eng/pipelines/coreclr/templates/run-superpmi-asmdiffs-job.yml

```diff
@@ -92,7 +92,6 @@ jobs:
         BuildConfig: ${{ parameters.buildConfig }}
         osGroup: ${{ parameters.osGroup }}
         archType: ${{ parameters.archType }}
-        shouldContinueOnError: true # Run the future step i.e. upload superpmi logs
```

Comment:
> Note that asm diffs should not be treated as failures.

However, I still see that you changed the asmdiff file as well? If there are asmdiffs, do you remember if the script's exit code is `0`. If yes, then your change seems good. We will only show it failed when anything really fails during asmdiff.